src/python/pants/option/global_options.py

```diff
@@ -361,7 +361,8 @@ def register_bootstrap_options(cls, register):
 
     register('--remote-execution', advanced=True, type=bool,
              default=DEFAULT_EXECUTION_OPTIONS.remote_execution,
-             help="Enables remote workers for increased parallelism. (Alpha)")
+             help="Enables the remote store and remote build execution for increased "
```

Comment:
I think the original help was better. From the user perspective you only turn on remote execution or not. The fact there is a seperate setting for CAS and executor is an un-neccessary detail to wade into here.

Reply:
Sounds good. Thanks for the review!